cf/cmd/cli/commands/provider/provider.go

```diff
@@ -19,11 +30,72 @@ var BootstrapCommand = cli.Command{
 	Name:        "bootstrap",
 	Description: "bootstrap a provider from the registry",
 	Usage:       "bootstrap a provider from the registry",
+	Flags: []cli.Flag{
+		&cli.StringFlag{Name: "bootstrapBucket", Aliases: []string{"bb"}, Usage: "The name of the bootstrap bucket provider is deployed to", EnvVars: []string{"DEPLOYMENT_BUCKET"}},
+	},
 	Action: func(c *cli.Context) error {
 		id := c.Args().First()
 		if id == "" {
 			return errors.New("id argument must be provided")
 		}
+
+		var cfg config.ProviderDeploymentCLI
+		ctx := context.Background()
+		_ = godotenv.Load()
+
+		err := envconfig.Process(ctx, &cfg)
+		if err != nil {
+			return err
+		}
+
+		registryClient, err := providerregistrysdk.NewClientWithResponses(cfg.ProviderRegistryAPIURL)
+		if err != nil {
+			return errors.New("error configuring provider registry client")
+		}
+
+		keys := strings.Split(id, "@")
+
+		if len(keys) != 2 {
+			return errors.New("incorrect provider id given")
+		}
+
+		version := keys[1]
+
+		teamAndName := strings.Split(keys[0], "/")
+
```

Comment:
You should add another length check here.
I think we will want to add a parser that can parse a provider string and return an object containing the components